docs/src/index.md

```diff
@@ -4,17 +4,19 @@
 <img src="assets/complex_plot.png" width="300" align="right"/>
 ```
 
-This is the documentation of the UnfoldMakie.jl module (aka library) for the Julia programming language. 
+This is the documentation of the UnfoldMakie.jl package for the Julia programming language. 
 
-## Benefits of UnfoldMakie.jl
+## Highlights of UnfoldMakie.jl
 
 - **10 plot functions for displaying ERPs.**
 Each plot emphasizes certain dimensions while collapsing others.
+- **Fast plotting**
+Plot one figure with 20 topoplots in 1 second? No problemo!
 - **Highly adaptable.**
-The module is based on the [Unfold](https://github.com/unfoldtoolbox/unfold.jl/) and [Makie](https://makie.juliaplots.org/stable/) modules, so you can use configurations from these modules to add new features to your figures.
+The package is primarily based on [Unfold.jl](https://github.com/unfoldtoolbox/unfold.jl/) and [Makie.jl](https://makie.juliaplots.org/stable/).
```

Comment:
Actually, the link to unfold is only in Designmatrix right? Because we could put them unfold as a submodule and wouldn't even require it for most plots

Reply:
What do you mean "the link to unfold "? Like require designmatrix or some other function from Unfold? plot_erp requires also coeftable, plot_topoplotsries needs eeg_matrix_to_dataframe
Does it counts?

Reply:
plot_erp requires coeftable? it requires a tidy dataframe, but not coeftable right?
you cat do plot_erp(uf::UnfoldModel) or can you?

and eeg_matrix_to_dataframe is in eegseries.jl in unfoldmakie

Reply:
exactly, whether you need a type or function from Unfold anywhere except designmatrix

Reply:
for plot_erp is plot_data::DataFrame;
sooo only plot_dm requires designmatrix()

Reply:
ok, I will write an issue - this is nice, we'll loose a dependency :-)

Also e.g. UnfoldSim.jl has no dependency on Unfold, I think this is a good thing. We still need to have the nicer interface of calling plot_erp with e.g. a matrix; or is this already implemented?

Reply:
In test i dont have any such example yet